### ISSUE TITLE
Accuracy loss from batch size of 256

### ISSUE THREAD

**Issue by xsacha** [ZhaoJ9014/face.evoLVe.PyTorch#59]
I have 2 x Quadro GV100 and they don't have enough memory to run training at a batch size of 512 for celebrity dataset with IR_100. I was wondering if there is a noticeable loss of accuracy when training this model for a batch size of 256?

Edit: accuracy was 99.15% on LFW after first epoch.

What cards do you recommend? I assume I need around 82GB of GPU memory.

**Comment by ZhaoJ9014** [comment]
Hi, there should not be a noticeable loss of accuracy for a smaller batch size. You can train your models for more epochs and see. Currently, I am using NVIDIA RTX 2080.
